docs/remote-interface.md

```diff
@@ -77,3 +77,17 @@ $ make BUILD_REMOTE=1 BUILD_DEBUG=1 check
 
 ## Requirements
 Remote interface requires RabbitMQ broker
+
+## Fallback mechanism
+If the remote interface is not working for some reason, the PoW calculation will be transferred to the local hardwares.\
```

Comment:
I have no clue why you would have to add `\` manually? Markdown viewer shall be always responsible for appropriate layout.

Reply:
Sometimes I do not want the rendered line to be too long.
Hence I break the line manually for a better visualization in my point of view.

Shall I remove them all?
I added it in the other documents, too.

Reply:
If so, I would like to remove them in the new pull request, which is for removing `\` in the documents and revise the source code to fit the `clang format`.

Reply:
Contents are more important than formatting. Go ahead.

Reply:
I'll remove the `\` symbol added in this pull request.
The rest of them in other documents will be removed in the next new pull request.

Reply:
Done.